.github/workflows/bump-version.yml

```diff
@@ -35,7 +35,7 @@ jobs:
   bump-version:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8  # v3.1.0
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
any reason we use the hash here and the more readable version elsewhere?

Reply:
I just applied the diff from the [closed PR](https://github.com/getsentry/snuba/pull/4879) created by dependabot and so do not really know the answer. My guess is that dependabot is just following the pattern of what was done in the past. Do you have a strong preference of one vs. other? I tend to download the patch from dependabot PR and `git apply /path/to/patch-file` avoiding any changes of my own doing in these bump PRs.

Why do I have to manually apply the patch and why can't dependabot PRs pass CI is another question that I and @mdtro discussed elsewhere.